src/config/babelrc.js

```diff
@@ -81,7 +81,7 @@ module.exports = () => ({
       ? require.resolve('babel-plugin-transform-inline-environment-variables')
       : null,
     [require.resolve('@babel/plugin-proposal-class-properties'), {loose: true}],
-    require.resolve('babel-plugin-minify-dead-code-elimination'),
+    ['babel-plugin-minify-dead-code-elimination', {keepFnName: isTest}],
```

Comment:
Let's add:

```js
const isMinify = parseEnv('BUILD_MINIFY', false)
```

And then change this to:

```suggestion
    isMinify ? require.resolve('babel-plugin-minify-dead-code-elimination') : null,
```

Reply:
Done.  Also, I didn't even notice I have removed the `require.resolve` in my first commit 😅 

P.S. I would have taken your suggestion if it were possible to add the `parseEnv` at the same time.